WebJobs.proj

```diff
@@ -294,4 +322,23 @@
       </Code>
     </Task>
   </UsingTask>
+  
+  <UsingTask TaskName="ReplaceFileText" TaskFactory="CodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v4.0.dll">
+  <ParameterGroup>
+    <FileName ParameterType="System.String" Required="true" />    
+    <MatchExpression ParameterType="System.String" Required="true" />
+    <ReplacementText ParameterType="System.String" Required="true" />
+  </ParameterGroup>
+  <Task>
+    <Reference Include="System.Core" />
+    <Using Namespace="System" />
+    <Using Namespace="System.IO" />
+    <Using Namespace="System.Text.RegularExpressions" />
```

Comment:
The way the TeamCity jobs worked was that all the repos requiring this common behavior pulled WebJobs.proj from a common repo http://vstfrd:8080/Azure/One/_git/AzureJobsInternal. If you look there, all the logic you're adding in this PR is in that repo.

It'd be good I think if we had one version of this to maintain, rather than copies in each repo? That'd mean a shared public repo with the common build stuff.

Reply:
That's a good idea. I've filed https://github.com/Azure/azure-webjobs-sdk/issues/1610 so we (or I :-)) can tackle that in the future.